docs/ui/interactions/fragments/web/chatbot/usage.md

```diff
@@ -0,0 +1,238 @@
+## Usage
+
+<docs-filter framework="react">
+
+```jsx
+import React from "react";
+import Amplify from "aws-amplify";
+import {AmplifyChatbot} from "@aws-amplify/ui-react";
+import awsconfig from "./aws-exports";
+
+Amplify.configure(awsconfig);
+
+const App = () => (
+  <AmplifyChatbot
+    botName="yourBotName"
+    botTitle="My ChatBot"
+    welcomeMessage="Hello, how can I help you?"
+  />
+);
+```
+
+</docs-filter>
+
+<docs-filter framework="angular">
+
+_app.module.ts_
+
+```js
+import {BrowserModule} from "@angular/platform-browser";
+import {NgModule} from "@angular/core";
+import {AppComponent} from "./app.component";
+
+import {AmplifyUIAngularModule} from "@aws-amplify/ui-angular";
+import Amplify from "aws-amplify";
+import awsconfig from "../aws-exports";
+
+Amplify.configure(awsconfig);
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [AmplifyUIAngularModule, BrowserModule],
+  providers: [],
+  bootstrap: [AppComponent],
+})
+export class AppModule {}
+```
+
+_app.component.html_
+
+```html
+<amplify-chatbot
+  bot-name="yourBotName"
+  bot-title="My ChatBot"
+  welcome-message="Hello, how can I help you?"
+/>
+```
+
+</docs-filter>
+
+<docs-filter framework="ionic">
+
+_app.module.ts_
+
+```js
+import {BrowserModule} from "@angular/platform-browser";
+import {NgModule} from "@angular/core";
+import {AppComponent} from "./app.component";
+
+import {AmplifyUIAngularModule} from "@aws-amplify/ui-angular";
+import Amplify from "aws-amplify";
+import awsconfig from "./aws-exports";
+
+Amplify.configure(awsconfig);
+
+@NgModule({
+  declarations: [AppComponent],
+  imports: [AmplifyUIAngularModule, BrowserModule],
+  providers: [],
+  bootstrap: [AppComponent],
+})
+export class AppModule {}
+```
+
+_app.component.html_
+
+```html
+<amplify-chatbot
+  bot-name="yourBotName"
+  bot-title="My ChatBot"
+  welcome-message="Hello, how can I help you?"
+/>
+```
+
+</docs-filter>
+
+<docs-filter framework="vue">
+
+_main.js_
+
+```js
+import Vue from "vue";
+import App from "./App.vue";
+import "@aws-amplify/ui-vue";
+import Amplify from "aws-amplify";
+import awsconfig from "./aws-exports";
+
+Amplify.configure(awsconfig);
+
+new Vue({
+  render: (h) => h(App),
+}).$mount("#app");
+```
+
+_App.vue_
+
+```html
+<template>
+  <amplify-chatbot
+    bot-name="yourBotName"
+    bot-title="My ChatBot"
+    welcome-message="Hello, how can I help you?"
+  />
+</template>
+```
+
+</docs-filter>
+
+<inline-fragment framework="react" src="~/ui/interactions/fragments/web/chatbot/properties-jsx.md"></inline-fragment>
```

Comment:
I remember there was a limitation to a nested `inline-fragment` like this. Is this working properly?

Reply:
Yes, this was working properly on local.